e2e/test/scenarios/native/reproductions/16584-run-selected-text-does-not-preserve-parameter-values.cy.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { restore, openNativeEditor, runNativeQuery } from "e2e/support/helpers";
+
+describe("issue 16584", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+    cy.intercept("POST", "/api/dataset").as("dataset");
+  });
+
+  it("should pass parameters when running with 'Run select text' (metabase#16584)", () => {
+    // The bug described in is #16584 can be further simplified:
+    // - the issue persists even when selecting the *entire* query
+    // - the issue is unrelated to using a date filter, using a text filter works too
+    // - the issue is unrelated to whether or not the parameter is required or if default value is set
+    openNativeEditor()
+      .type("SELECT * FROM ACCOUNTS WHERE COUNTRY = {{ country }} ", {
```

Comment:
Nit:
You don't have to select all results for this particular repro. Cypress is already heavy on the resources.
I'd maybe go with `select country from...` or `select count(*)`. One additional, and very compelling reason could be that some tables have so many columns that some of them will be rendered outside of a viewport.

And since we use virtualization in the table, you might end up having to scroll.
Not saying it's the case here. Just sharing some of the traps I've ran into before.

In such cases, it's perfectly fine to limit the number of columns.